src/Silk.Core/Utilities/ExperimentalAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Linq;
+using DSharpPlus.CommandsNext;
+
+namespace Silk.Core.Utilities
+{
+    ///<summary>
+    /// Denotes this command is experimental, and may not work properly.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class ExperimentalAttribute : Attribute { }
+
+    public static class ExperimentalAttributeExtensions
+    {
+        public static bool IsExperimental(this Command c) =>
+            c.CustomAttributes.OfType<ExperimentalAttribute>().Any();
```

Comment:
Definitely a better way to go about this

Reply:
Could move the Attribute to the Extensions project. Reason I didn't move it was in case there were things the attribute needed which were in Silk.Core, since Silk.Extensions project doesn't reference Silk.Core

Reply:
I was moreso agreeing that "[This is] definitely a better way to go about this", not "[There is] definitely a better.."

Reply:
Ohh, lol! Sorry! Thank you! 👍🏽